portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,17 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+var modal = document.getElementById("myModal");
+var i;
+var img = document.getElementsByClassName("gallery");
+var modalImg = document.getElementById("img01");
+ for(i=0;i< img.length;i++)
```

Comment:
Can you use this block style: https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks? (and anywhere else it's relevant)

Reply:
I don't really get it. Would that mean a space before and after the for loop?

Reply:
Yeah so it would be a space after `for` and remove the newline before the `{`, like this:
`for (i = 0; i < img.length; i++) {`

Reply:
thanks!